Rakefile

```diff
@@ -12,7 +12,7 @@ task :test do
     :allow_hash_href => true,
     :check_html => true,
     :file_ignore => [/javadoc/, /v2/, /pydoc/],
-    :url_ignore => [/jstorm.io/, /datatorrent.com/],
+    :url_ignore => [/jstorm.io/, /datatorrent.com/, /ai.google/],
```

Comment:
Can you cite https://issues.apache.org/jira/browse/INFRA-16527 for removal of ai.google?

Reply:
Done